### ISSUE TITLE
nn.AvgPool2d(kernel_size=3, stride=2, padding=1) torchsharp not support padding？

### ISSUE THREAD

**Issue by williamlzw** [dotnet/TorchSharp#870]
nn.AvgPool2d(kernel_size=3, stride=2, padding=1) torchsharp not support padding?

AvgPool2d(kernel_size: 3, stride: 2);

**Comment by williamlzw** [comment]
torch.nn.functional.avg_pool2d can temporarily solve this problem

**Comment by SteveJSteiner** [comment]
Hi William, 
Thank you for (another!) temporary workaround. [NiklasGustafsson](https://github.com/NiklasGustafsson) asked me to keep an eye out and at minimum let contributors know he is away until early next year.

**Comment by NiklasGustafsson** [comment]
It is odd that this was missed... Should be easy to fix.
